.github/workflows/deploy-2-start.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           spack env activate ${{ inputs.env-name }}
           spack --debug install --fresh || exit $?
           spack module tcl refresh --delete-tree -y
-          spack find --paths > ${{ vars.SPACK_LOCATION }}/var/spack/environments/${{ inputs.env-name }}/spack.location
+          spack find --format '{name}@{version} {prefix}' | jq -Rn '[inputs | split(" ") | {(.[0]):(.[1])}] | add' > ${{ vars.SPACK_LOCATION }}/var/spack/environments/${{ inputs.env-name }}/spack.location
```

Comment:
It is great to have a machine-readable version. Can (should) we add a `.json` suffix? Would that make it open in a browser if clicked, or does it always force a download?

What do you think about having both?

Reply:
Good point. Lets do both!